src/ui/index.js

```diff
@@ -101,7 +101,10 @@ export default async (kbnServer, server, config) => {
           includeUserProvidedConfig
         }),
         bundlePath: `${config.get('server.basePath')}/bundles`,
+        // #{i18n('KEY')} -> value ("Hi!")
         i18n: key => _.get(translations, key, ''),
+        // var v = #{i18n_var('KEY')} -> var v = "value (\"Hi!\")"
+        i18n_singleQuoteVar: key => _.get(translations, key, '').replace(/'/g,'\\\''),
       });
```

Comment:
Not sure we would want 2 variables passed which contain the separate instances of the translations. Refer to comment in  'src/ui/views/ui_app.jade'  by using double quotes.